reddit_about/__init__.py

```diff
@@ -46,10 +46,11 @@ class About(Plugin):
     }
 
     def add_routes(self, mc):
-        # handle wildcard after /about/:action/ for postcard pushState URLs.
-        for route in ('/about/:action', '/about/:action/*etc'):
-            mc(route, controller='about', conditions={'function':not_in_sr},
-               requirements={'action':'team|postcards|alien|values'})
+        mc('/about/postcards', controller='about', action='postcards',
+           conditions={'function':not_in_sr})
```

Comment:
And one more tiny thing - these should be indent by 4, not 3 (so they'll line up with `dest` on L55.